src/hapi/pipelines/app/__main__.py

```diff
@@ -64,12 +64,38 @@ def parse_args():
     return parser.parse_args()
 
 
+def combine_default(country, default):
+    country["input"] = country["input"] + default["input"]
+    country["output"] = country["output"] + default["output"]
+    country["output_hxl"] = country["output_hxl"] + default["output_hxl"]
+    return country
+
+
+def inject_default(config):
+    if "default" in config:
+        default = config["default"]
+        keys = default["keys"]
+        if len(keys) > 0:
+            for key in keys:
+                for country_key in config[key]:
+                    country = config[key][country_key]
+                    combine_default(country, default)
+        else:
```

Comment:
Do I understand correctly that this structure is to handle the different possible "depths" of the scrapers, e.g. population vs operational_presence? I'm not sure if it's needed - I think @b-j-mills and @mcarans are working on getting the operational_presence scrapers into a similar format (but I may be mistaken).

Reply:
Yes, that's correct, the two current configs were different. I will check what the plan is for operational presence

Reply:
I just checked and thanks to Briar and Mike's recent changes, they are now on the same level, i.e. you can remove everything under `else`